components/datepicker/package.json

```
@@ -19,17 +19,16 @@
   },
   "peerDependencies": {
     "@spectrum-css/pickerbutton": ">=3",
-    "@spectrum-css/popover": ">=5 <=6",
-    "@spectrum-css/vars": ">=9"
+    "@spectrum-css/popover": ">=5"
```

```suggestion
    "@spectrum-css/popover": ">=6"
```

Should this use v6 instead of 5?

---

We'll need to add the tokens package here too